modules/ROOT/pages/asset-details.adoc

```diff
@@ -0,0 +1,40 @@
+= Asset Details
```

Comment:
Since the asset portal is interactive and not just a display, I'm going to stick with "Asset Details" for now, but I am open to further ideas and suggestions.